fingerprint/infra/scanner/src/main/java/com/simprints/fingerprint/infra/scanner/v2/scanner/ScannerInfo.kt

```diff
@@ -0,0 +1,56 @@
+package com.simprints.fingerprint.infra.scanner.v2.scanner
+
+
+/**
+ * Singleton class to manage and provide access to scanner-related information.
+ *
+ * This class handles two key pieces of information:
+ * 1. `scannerId`: The unique identifier of the scanner, available after connecting to the scanner.
+ * 2. `un20SerialNumber`: The serial number of the scanner, available only after reading an unprocessed image.
+ *
+ * Both properties can be updated independently based on the scanner's lifecycle and operations.
+ *
+ * Todo: When refactoring this module to use DI, consider using Hilt annotations (e.g., `@Singleton` and `@Inject`) to manage this class as a singleton within the DI framework.
```

Comment:
If this is to be done later, it would be good to have a ticket for it so this doesn't add to "forgettable" todos.

Reply:
Thanks, @alex-vt   
I've created a ticket to implement DI for the entire scanner module: [MS-835](https://simprints.atlassian.net/browse/MS-835)

Reply:
Why not do it in this PR, tho? This is in the main, so there is no rush with the implementation.

Reply:
I refactored the scanner module to use Hilt for dependency injection.